include/display/grove_lcd.h

```diff
@@ -11,6 +11,20 @@
 extern "C" {
 #endif
 
+/**
+ * @brief Networking
```

Comment:
Looks like a copy-paste typo to  me, considering which file this is in (and the line below this)

Reply:
bah